docs/src/increase-observability/logs/forward-logs.md

```diff
@@ -0,0 +1,123 @@
+
+View your logs in the **Logs** dashboard.
+
+<--->
+---
+title=Splunk
+highlight=false
+file=none
+---
+
+1. In Splunk, get an Event Collector token on [Splunk Platform](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector#Create_an_Event_Collector_token_on_Splunk_Cloud_Platform)
+   or [Splunk Enterprise](https://docs.splunk.com/Documentation/Splunk/latest/Data/UsetheHTTPEventCollector#Create_an_Event_Collector_token_on_Splunk_Enterprise).
+2. Determine your host, which is the Splunk instance that's collecting data.
+3. Choose an index name.
+4. Create the integration with the following command:
+
+   <div class="highlight"><pre tabindex="0" class="chroma"><code class="language-bash" data-lang="bash"><span class="line"><span class="cl">platform project:curl /integrations -X POST -d <span class="s1">'{ "type":"splunk", "url":"https://http-inputs.{{< variable "HOST" >}}.splunkcloud.com/services/collector/event", "index":"{{< variable "HOST" >}}", "token":"{{< variable "TOKEN" >}}" }'</span></span></span></code></pre></div>
+
+View your logs in the **Apps->Search & Reporting** dashboard.
+Filter by the index name to find the relevant events.
+
+<--->
+---
+title=Sumo Logic
+highlight=false
+file=none
+---
+
+1. In Sumo Logic, [configure an HTTP source](https://help-opensource.sumologic.com/docs/send-data/hosted-collectors/http-source/logs-metrics/#configure-an-httplogs-and-metrics-source).
+   Make sure to copy the Source Category and collector URL.
+2. Create the integration with the following command:
+
+   <div class="highlight"><pre tabindex="0" class="chroma"><code class="language-bash" data-lang="bash"><span class="line"><span class="cl">platform project:curl /integrations -X POST -d <span class="s1">'{ "type":"sumologic", "url":"{{< variable "COLLECTOR_URL" >}}", "category":"{{< variable "SOURCE_CATEGORY" >}}" }'</span></span></span></code></pre></div>
+
+View your logs in the **Log Search** tab.
+
+{{< /codetabs >}}
+
+Logs start getting forwarded automatically after the integration has been resolved,
+which can take up to an hour.
+
+## Forward to a syslog endpoint
+
+Syslog is a standard for message logging.
+Many third-party services offer endpoints for ingesting syslog events.
+You can forward your Platform.sh logs to any of those endpoints.
+
+To enable this forwarding, run a command like the following:
+
+```bash
+platform project:curl /integrations -X POST --data '{ "type":"syslog", "host":"{{< variable "INGESTION_HOST" >}}", "port":"{{< variable "INGESTION_HOST_PORT" >}}" }'
+```
+
+`type`, `host`, and `port` are the only properties required for all endpoints.
+The following table shows the other available properties:
+
+| Property         | Type      | Default    | Description |
+| -------- --------| --------- | ---------- | ----------- |
```

Comment:
🐞 There is a display issue with the table in the output. I think it might be because of a missing hyphen.